### PR TITLE
Translated the front readme file

### PR DESCRIPTION
**Pull request by TomasRoj**
I am not sure if I did everything correctly so I am open to any questions or rewievs. Hope this helps!

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
+Díky, za investování tvého času do Arduino obsahu. Je to opravdu přínosné pro celou (celosvětovou) Arduino komunitu. Pokud se chceš dozvědět víc o přispívání do tohoto projektu, prosím navštiv  https://create.arduino.cc/projecthub/Arduino_Genuino/contribute-to-the-arduino-reference-af7c37[tento tutoriál "krok po kroku"].
 
-== Content Creation and Editing
-If you want to contribute new content, create a new file (with any text or code editor) and save it as .adoc.  
-Do not use parenthesis or any special character in the file name.  
-In https://raw.githubusercontent.com/arduino/reference-cz/master/AsciiDoc_sample/AsciiDoc_Dictionary/AsciiDoc_Template-Dictionary.adoc[`reference-cz/AsciiDoc_sample/AsciiDoc_Dictionary`] you will find an overview on the AsciiDoc syntax. This includes Titles, Text, Links, Images, Tables, Code and various embeds (video, slideshow, audio and code). You can see it rendered https://www.arduino.cc/reference/cz/asciidoc_sample/asciidoc_dictionary/asciidoc_template-dictionary/[here].
```
Comment (per1234) [inline README.adoc:11]: You forgot to translate this line. You only deleted it.

File: README.adoc [hunk 2]
```diff
@@ -1,37 +1,35 @@
...
+* Nebo https://raw.githubusercontent.com/arduino/reference-cz/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Parent_Of_Entities.adoc[`AsciiDoc_Template-Parent_Of_Entities.adoc`] (poskytnuty https://www.arduino.cc/reference/cz/asciidoc_sample/reference_terms/asciidoc_template-parent_of_entities/[zde]) proskupiny funkcí jako link:http://arduino.cc/en/Reference/Serial[Serial].
 
-If you need to add images to the Asciidoc please create a folder called attachments in the same directory as the Asciidoc file. Images can be saved in SVG and PNG format, max size 200KB.
+Prosím pamatuj, že každý soubor s dokumenací by měl obsahovat minimálně popis, příklad kódu a linky na ostatní  relevantní zdroje (Podívej se také do sekce "Také")
+Pokud potřebuješ přidat obrázky do ASCII dokumentu, prosím vytvoř složku pojmenovanou "attachments" ve stejné složce, jako je ASCII dokument. Obrázku mohou být uloženy v SVG nebo PNG formátu (ne JPG!), s maximální velikostí 200KB.
```
Comment (per1234) [inline README.adoc]: Don't translate the word "Asciidoc". This is a name of a markup format and so the word is the same in any language. You'll need to be careful about this sort of thing throughout your translations. For example, you would not want to translate code syntax. You should translate string literals and comments in the code. You can translate variable names in the example code if you like, though you should make sure to translate all occurrences of the variable name in the code.

**Review by per1234 — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (TomasRoj) [reply README.adoc]: Ok, fixed

File: README.adoc (hunk 1 above)
Comment (TomasRoj) [reply README.adoc:11]: Ok, this too

File: README.adoc (hunk 2 above)
Comment (per1234) [reply README.adoc]: You added the translation of this line in the wrong place in the text. It should be part of the paragraph before the bullet points.

File: README.adoc (hunk 1 above)
Comment (per1234) [reply README.adoc:11]: You added the translation of this line in the wrong place in the text. It should be part of the paragraph before the bullet points.

**Comment by TomasRoj** [comment]
I don't understand what do you mean by this

File: README.adoc (hunk 1 above)
Comment (per1234) [reply README.adoc:11]: Here is the position of the sentence before the translation, in the paragraph before the bullet points:

---
en

---
Here is the position of the sentence after the translation, after the bullet points:

---
cs

---

Comment (TomasRoj) [reply README.adoc:11]: Ok thanks for your drawings! I understand now.

**Comment by TomasRoj** [comment]
Is now everything ok?

**Review by per1234 — APPROVED** [review]

**Comment by per1234** [comment]
Thanks so much @TomasRoj! Great work.

**Comment by TomasRoj** [comment]
Thx. I cannot believe that I made contribution to official arduino account 😂

**Pull request merged**